cfgov/unprocessed/css/print.less

```diff
@@ -90,35 +92,135 @@
         // These pseudo-classes are needed to guarantee specificity
         &:link,
         &:visited {
-            border: 0;
+            // Convert stand-alone (jump) links to standard inline links
+            border-top: 0;
+            display: inline;
             padding-top: unit( 5px / @base-font-size-px, em );
-            padding-bottom: unit( 5px / @base-font-size-px, em );
+            padding-bottom: 0;
         }
     }
 
+    .m-list__links .m-list_item {
+        // Space out stand-alone links by .5em
+        margin-bottom: unit( 8px / @base-font-size-px, em );
+    }
+
+    .o-info-unit-group {
+        .content-l_col-1-2,
+        .content-l_col-1-3,
+        .content-l_col-2-3 {
+            // Ensure content remains vertically aligned by rendering them as a table cell
+            display: table-cell !important;
+        }
+        .content-l_col-1 {
+            .grid_column( 12 );
+        }
+        .content-l_col-1-2 {
+            // Keep 50/50 info unit groups at two columns for print
+            .grid_column( 6 );
+        }
+        .content-l_col-1-3 {
+            // Keep 33/33/33 info unit groups at three columns for print
+            .grid_column( 4 );
+        }
+        .content-l_col-2-3 {
+            // Maintain 2/3 layouts for print
+            .grid_column( 8 );
+        }
+        .m-info-unit_image {
+            // Print info unit images
+            display: block;
+        }
+    }
+
+    // Force FCMs into multiple columns when printing
+    .o-featured-content-module {
+        display: flex;
+
+        &_text {
+            flex: 1;
+            padding-right: @fcm-visual-width + @grid_gutter-width;
+            padding-left: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+
+        &_visual {
+            height: 100%;
+            overflow: hidden;
+            padding-right: 0;
+            padding-bottom: 0;
+            padding-left: 0;
+            position: absolute;
+            top: 0;
+            right: 0;
+            width: @fcm-visual-width;
```

Comment:
For better organization, maybe move `width` up so it sits next to `height`.

Reply:
Swapped around in https://github.com/cfpb/consumerfinance.gov/pull/6369/commits/f70ca17ebb9c81b68c944295277b4110d4c42695